composer.json

```
@@ -3,8 +3,8 @@
     "description": "Serve files to the browser that are not available directly",
     "type": "library",
     "require": {
-        "php": "^5.3.9 | ^7.0",
-        "symfony/http-foundation": "^2.8|^3.0"
+        "php": "^5.3.9 | ^7.0 | ^8.0",
+        "symfony/http-foundation": "^2.8|^3.0|^4.0|^5.0"
```

Could do without unsupported Symfony-versions...
```suggestion
        "symfony/http-foundation": "^4.4|^5.4"
```
and maybe this also works on Sf 6?
```suggestion
        "symfony/http-foundation": "^4.4|^5.4|^6.0"
```